articles/README.md

```diff
@@ -41,6 +41,35 @@ for contributors:
 - The titles of the articles, the names in the TOC and the *uid* of the metadata
   should be as close as possible among them and represent clearly the H1 header
   of the Markdown document.
+  
+## Broken links prevention
+
+The URL of article pages on `docs.microsoft.com` is generated from the relative path to the `articles` folder on the GitHub repository. For example, the article located at `./articles/user-guide/programs.md` is assigned the URL `https://docs.microsoft.com/azure/quantum/user-guide/programs`. This fact means that each time an article changes its path, either because of a relocation or a change in the file name, the URL will change accordingly. This change can cause some users to find broken links to previously existing articles. To avoid this:
+
+- Try to not rename or relocate articles.
+- If for some reason you need to rename, remove or relocate an article, you should add a redirect from the previous article to the new URL to avoid creating any broken links.
```

Comment:
```suggestion
- If for some reason you need to rename, remove or relocate an article, you need to add a redirect from the previous article to the new URL to avoid creating any broken links.
```